src/Reflow/Parser.hs

```diff
@@ -69,7 +69,7 @@ mappend3 :: (Monoid m) => m -> m -> m -> m
 mappend3 a b c = a <> b <> c
 
 singleLine :: Parser Text
-singleLine = pack <$> manyTill anyChar (try eol)
+singleLine = pack <$> manyTill anyChar (try trailingWhitespace >> eol)
```

Comment:
I meant the latter. The tests currently pass but I'll update to make sure that this is working how I expect.